### PR TITLE
updating links from fwlink to real link if exists

### PR DESCRIPTION
**Pull request by eddynaka**
updating links from fwlink to real link if exists - contributes to #3391

### CODE REVIEW

**Review by Thraka — APPROVED** [review]
Mine look good.

**Comment by mairaw** [comment]
Please don't merge. I have several comments still to submit.

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @eddynaka. Left some suggestions.

File: docs/framework/wpf/graphics-multimedia/how-to-create-a-shape-by-using-a-pathgeometry.md [hunk 1]
```diff
@@ -21,11 +21,11 @@ A triangle created with a PathGeometry
   
  The previous example showed how to create a relatively simple shape, a triangle. A <xref:System.Windows.Media.PathGeometry> can also be used to create more complex shapes, including arcs and curves. For examples, see [Create an Elliptical Arc](how-to-create-an-elliptical-arc.md), [Create a Cubic Bezier Curve](how-to-create-a-cubic-bezier-curve.md), and [Create a Quadratic Bezier Curve](how-to-create-a-quadratic-bezier-curve.md).  
   
- This example is part of larger sample; for the complete sample, see the [Geometries Sample](https://go.microsoft.com/fwlink/?LinkID=159989).  
+ This example is part of larger sample; for the complete sample, see the [Geometries Sample](https://github.com/Microsoft/WPF-Samples/tree/master/Graphics/Geometery).
```
Comment (mairaw) [inline docs/framework/wpf/graphics-multimedia/how-to-create-a-shape-by-using-a-pathgeometry.md]: Can you review this link to go to https://github.com/Microsoft/WPF-Samples/tree/master/Graphics/Geometry everywhere? I just submitted a PR to fix that on that repo

Comment (eddynaka) [reply docs/framework/wpf/graphics-multimedia/how-to-create-a-shape-by-using-a-pathgeometry.md]: ok!

Comment (eddynaka) [reply docs/framework/wpf/graphics-multimedia/how-to-create-a-shape-by-using-a-pathgeometry.md]: just added 3 more files with the same typo!

**Review by mairaw — APPROVED** [review]
Thanks @eddynaka!

**Pull request merged**